site/app/templates/forum/CreatePost.twig

```diff
@@ -32,7 +32,7 @@
 
     {% endif %}
 
-    <pre class='pre_forum'><span class="post_content pre-wrap" >{% include "forum/RenderPost.twig" with { "post_content": post_content, "render_markdown": render_markdown } only %}</span></pre>
+    <pre class='pre_forum'><span class="post_content markdown" style="white-space: normal;" >{% include "forum/RenderPost.twig" with { "post_content": post_content, "render_markdown": render_markdown } only %}</span></pre>
```

Comment:
Ah, good catch. I thought the markdown button was just for displaying the shortcut buttons for markdown, but that makes much more sense that it can be toggled, not always using markdown. I will look into these changes.